mpmp/utilities/analysis_utilities.py

```diff
@@ -100,7 +103,33 @@ def load_purity_binarized_results(results_dir):
     return results_df
 
 
-def load_purity_by_cancer_type(results_dir, sample_info_df):
+def load_purity_results(results_dir):
+    """Load results of tumor purity regression experiments.
```

Comment:
Does this function load regression results only?

Reply:
Currently, yes, but I think I can combine it with the function above (which does essentially the same thing for classification results), and just pass a classify flag to differentiate.